infra/src/main/java/com/redhat/service/bridge/infra/k8s/K8SBridgeConstants.java

```diff
@@ -0,0 +1,7 @@
+package com.redhat.service.bridge.infra.k8s;
+
+public class K8SBridgeConstants {
+    public static final String METADATA_TYPE = "type";
+    public static final String PROCESSOR_TYPE = "processor";
```

Comment:
Any reason why these can't be on the `ProcessorCustomResource` and `BridgeCustomResource`?